### PR TITLE
feat(adapter): rename config helper to `createConfig`

### PR DESCRIPTION
**Pull request by tanner-reits**
## What is the current behavior?

Helper is currently names `createStencilPlaywrightConfig` which is a bit wordy

GitHub Issue Number: N/A

## What is the new behavior?

Renames the helper to `createConfig`. The util is already in context/scope of the adapter package, so don't need to worry about confusion on its purpose. Developers can alias the import if they desire.

## Documentation

https://github.com/ionic-team/stencil-site/pull/1398

## Does this introduce a breaking change?

- [ ] Yes
- [X] No

Technically, it would be, but this is a pre-1.0 experimental package.

## Testing

Updated automated tests and confirmed tests execute correctly in a Stencil component starter project

### CODE REVIEW

**Review by alicewriteswrongs — APPROVED** [review]

**Review by christian-bromann — APPROVED** [review]
👍 

Alternatively we could keep `createStencilPlaywrightConfig` as an alias, deprecate it and remove it later on as a breaking change.

**Comment by tanner-reits** [comment]
@rwaskiewicz I created a breaking changes file to track these for now. This might go away depending on some release tooling discovery since those tools would be able to generate changelogs for us.
